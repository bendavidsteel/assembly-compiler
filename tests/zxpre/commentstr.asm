    mov %a,%b
    db "asd ; qwe"  ; A string with a semicolon in
