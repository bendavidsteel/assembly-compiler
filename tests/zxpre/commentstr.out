         mov %a,%b
         db "asd ; qwe"
